src/Tizen.NUI/src/public/BaseComponents/TextGeometry.cs

```diff
@@ -212,5 +216,137 @@ public static List<Position2D> GetTextPosition(TextLabel textLabel, int start, i
             CheckSWIGPendingException();
             return list;
         }
+
+        /// <summary>
+        /// Get the rendered size of the line based on a requested value, zero-based. <br />
+        /// if a line contains characters with different directions, multiple sizes will be returned for each block of contiguous characters with the same direction. <br />
```

Comment:
I wonder about return value, 
Could you explain more details with some example?
I don't know means of `different directions`
and each item of returning value is corresponding each characters?
and if it has same appearance, is only returning one value?